src/app/admin-page/admin-page.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="spaced-sides">
+  <div class="row">
+    <div class="col-4">
+      <app-student-list></app-student-list>
+    </div>
+    <div class="col-5">
+      <app-student-add-form *ngIf="(editOrCreateForm$ | async) === false"></app-student-add-form>
+      <app-student-edit-form *ngIf="editOrCreateForm$ | async"></app-student-edit-form>
+    </div>
+    <div class="col-3">
+      <app-stream-list></app-stream-list>
+    </div>
+  </div>
+</div>
```

Comment:
There is a better solution ;) 
```suggestion
      <ng-container *ngIf="(editOrCreateForm$ | async); then showStudentAddForm; else showStudentEditForm"></ng-container>
    </div>
    <div class="col-3">
      <app-stream-list></app-stream-list>
    </div>
  </div>
</div>
<ng-template #showStudentAddForm>
<app-student-add-form></app-student-add-form>
</ng-template>

<ng-template #showStudentEditForm>
 <app-student-edit-form></app-student-edit-form>
 </ng-template>
```